docs/docs/animations/withTiming.mdx

```diff
@@ -54,6 +54,26 @@ enum ReduceMotion {
 #### `toValue`
 
 The value on which the animation will come at rest.
+There are many possible types for `toValue`, currently the following kinds of values are supported:
+
+- **colors** - we support wide range of color formats:
+  - hex numbers - `0xff1234`
+  - rgb - `rgb(100,50,0)`
+  - rgba - `rgba(255,105,180,0)`
+  - hsl - `hsl(0, 50%, 50%)`
+  - hex strings - `#53575E`
+  - css color names - `dodgerblue`
```

Comment:
```suggestion
- **colors**
  - RGB (Red, Green, Blue) - e.g. `rgb(100, 50, 0)`,
  - RGBA (Red, Green, Blue, Alpha) - e.g. `rgba(255, 105, 180, 0)`,
  - RGB Hexadecimal - e.g. `#53575E`,
  - Hexadecimal integer literal - e.g. `0xff1234`,
  - HSL (Hue, Saturation, Lightness) - e.g.`hsl(0, 50%, 50%)`,
  - Named colors - e.g. `dodgerblue`.
```

let's use the official MDN naming

https://developer.mozilla.org/en-US/docs/Web/CSS/color_value
https://developer.mozilla.org/en-US/docs/Web/CSS/named-color
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#numeric_literals

Reply:
`0xff1234` is not a hexadecimal integer literal, but an actual number. Since even you misunderstood it I must have been not precise enough 😆 
Anyway, maybe we should skip it and stop supporting it (at least officialy) anyway? Jut because this is not supported by React-Native.